### PR TITLE
Feature/vta 531

### PR DESCRIPTION
**Pull request by Daniel-Searle**
## Premature Error Throwing Logic

To allow more than 1 non archived technical record but no more than 2
[link to ticket number](https://dvsa.atlassian.net/browse/VTA-531)

## Checklist

- [x] Code has been tested manually
- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/assets/Enums.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export enum HTTPRESPONSE {
     NO_STATUS_UPDATE_REQUIRED = "No status update required",
     NO_EU_VEHICLE_CATEGORY_UPDATE_REQUIRED = "No EU vehicle category update required",
     INVALID_EU_VEHICLE_CATEGORY = "Invalid EU vehicle category",
-    EU_VEHICLE_CATEGORY_MORE_THAN_ONE_TECH_RECORD = "The vehicle has more than one non archived Tech record.",
+    EU_VEHICLE_CATEGORY_MORE_THAN_TWO_TECH_RECORD = "The vehicle has more than two non archived Tech records.",
```
Comment (Ian-Hodges) [inline src/assets/Enums.ts:35]: I'm being picky, and it's not related to this change, but I think the naming could be improved. Does swapping 'non archived' to 'provisional' sound better? For example, this line would be:
`EU_VEHICLE_CATEGORY_MORE_THAN_TWO_PROVISIONAL_RECORDS = "The vehicle has more than two Provisional Tech records.`

Comment (Ian-Hodges) [reply src/assets/Enums.ts:35]: Ah, I assumed 'non archived' meant 'provisional'.

Comment (Ian-Hodges) [reply src/assets/Enums.ts:35]: You'll have to excuse my ignorance of this service. What happens if three records are returned? Won't that error prematurely still?

Comment (Ian-Hodges) [reply src/assets/Enums.ts:35]: Just read the comments on the Jira ticket. In the ticket description it has:
`If there are 2 provisionals and 1 current tech record then the latest provisional tech record should be shown on the app. `
Is this no longer the case and it should throw an error?

Comment (Daniel-Searle) [reply src/assets/Enums.ts:35]: It's been confirmed with Steve M that there we are only going to allow 2 non-archived tech records and any more, it should throw the error

Comment (Daniel-Searle) [reply src/assets/Enums.ts:35]: I'll speak to BA about correcting the description of the story @Ian-Hodges

**Review by cb-cs — APPROVED** [review]

**Pull request closed without merging**